HHS/OPRE-OPS
# Show Approval/Denial notifications for an Agreement

## What changed

When a user submits a change request that requires approval, when the approval/denial occurs and the original user views the Agreement they will see notifications of that approval or denial. 




## Issue
[OPS-2264](https://github.com/HHS/OPRE-OPS/issues/2264)

## How to test

Run python tests
Run cypress tests (new tests were added)
Run js tests (new tests were added)

Manually:
1. login as basic user
2. submit change request on Agreement 1/9
3. login as admin
4. approve change request
5. log back in as basic user
6. goto agreement details page for 1/9
7. see Approve or Decline Alert
8. 💰 profit

## Screenshots
Below is the notification shown when a user's change is approved:
https://github.com/user-attachments/assets/ceac2b75-465b-42d4-b003-f72a6ae31063

Below is the notification shown when a user's change is declined:
https://github.com/user-attachments/assets/129eefb0-58d0-4015-85a2-1fef6f1b7710

A screenshot of the notification center showing updated wording per initial UX discussions. The statuses are no longer all caps, underscore separated words.
<img width="1387" alt="Updated Notification Screenshot" src="https://github.com/user-attachments/assets/f08c4590-03b6-4a1f-8646-8f69ef128d9f">

## Definition of Done Checklist
